OracleGoldenGate/README.md

```diff
@@ -0,0 +1,98 @@
+      buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip --no-cache
+
+**IMPORTANT:** The result will be a Docker image with the Oracle GoldenGate binaries installed. Created images will follow the naming convention of **oracle/goldengate-&lt;edition&gt;:&lt;version&gt;**, for example:
+
+- `oracle/goldengate-standard:12.2.0.1.1`
+- `oracle/goldengate-standard:12.3.0.1.0`
+- `oracle/goldengate-microservices:12.3.0.1.0`
+
+#### Changing the Base Image
+By default, the base image used when building Oracle GoldenGate Docker images is `container-registry.oracle.com/database/instantclient:12.2.0.1`.  This value can be changed by setting the environment variable `BASE_IMAGE` when executing the `buildDockerImage.sh` script. For example:
+
+    BASE_IMAGE="container-registry.oracle.com/database/enterprise" ./buildDockerImage.sh ~/Downloads/fbo_ggs_Linux_x64_shiphome.zip
+
+### Running Oracle GoldenGate in a Docker container
+
+To run your Oracle GoldenGate Docker image use the **docker run** command as follows:
+
+    docker run --name <container name> \
+        -e OGG_SCHEMA=<schema for OGG> \
+        -e OGG_ADMIN=<admin user name> \
+        -e OGG_ADMIN_PWD=<admin password> \
+        -e OGG_DEPLOYMENT=<deployment name for Microservices Architecture> \
+        -v [<host mount point>:]/u02/ogg \
+        -v [<host mount point>:]/u02/ogg/var/data \
+        oracle/goldengate-microservices:12.3.0.1.0
+
+    Parameters:
+       --name:        The name of the container (default: auto generated)
+       -e OGG_SCHEMA: The GGSCHEMA to use for OGG (default: `oggadmin`)
+       -e OGG_ADMIN:  The name of the administrative account to create for Microservices Architecture (default: `oggadmin`)
+       -e OGG_ADMIN_PWD:
+                      The password for the administrative account (default: value of `OGG_ADMIN`)
+       -e OGG_DEPLOYMENT:
+                      The name of the deployment for Microservices Architecture (default: `Local`)
+       -v /u02/ogg
+                      The data volume for Microservices Architecture configuration data
+       -v /u02/ogg/var/data
+                      The data volume for Microservices Architecture trail data
+
+#### Administrative Account Password for Microservices Architecture
+
+On the first startup of a Microservices Architecture container, a random password will be generated for the Oracle GoldenGate administrative user if not provided. You can find this password at the start of the Docker container log:
+
+    docker logs <container name> | head -3
+    ----------------------------------------------------------------------------------
+    --  Password for administrative user 'oggadmin' is 'qVc3bqNlwijk'
+    ----------------------------------------------------------------------------------
+
+#### Running GGSCI in an OGG Standard Edition Docker container
+The **GGSCI** utility can be run in the OGG container with this command:
+
+    docker exec -ti --user oracle <container name> ggsci
```

Comment:
The above uses `docker exec`, but I think your comments apply instead to `docker run`.

Oracle GoldenGate comes with a command line utility:

- `ggsci` for Standard Edition
- `adminclient` for Microservices Architecture

The `ggsci` utility *must* be run from within the container because of on-disk state information shared with the background processes and other reasons.

The `adminclient` could be run from another container, but it seemed better to provide the same `docker exec` mechanism for both the Standard and Microservices editions.